src/scene/batching.js

```diff
@@ -0,0 +1,837 @@
+pc.extend(pc, function () {
+
+    /**
+     * @name pc.Batch
+     * @class Holds information about batched mesh instances. Created in BatchManager.create
+     * @property {Array} origMeshInstances An array of original mesh instances, from which this batch was generated.
+     * @property {pc.MeshInstance} meshInstance A single combined mesh instance, the result of batching.
+     * @property {pc.Model} model A handy model object, ready to use in Scene.addModel and Scene.removeModel
```

Comment:
Does this property need to be deleted from the docs? It's not in the code.

Reply:
Which property? Github doesn't make it clear to which line you point. If you mean the model property - it is going to be used by user.